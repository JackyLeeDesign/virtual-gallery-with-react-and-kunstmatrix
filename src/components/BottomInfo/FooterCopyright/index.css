.footer_copyright {
    background: #6785c1;
    padding: 10px;
    line-height:1.5em;
    color: #e6e6e6;
    text-align: center;
}

