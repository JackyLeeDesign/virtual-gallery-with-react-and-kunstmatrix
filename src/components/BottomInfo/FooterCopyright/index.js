import React from "react";
import style from "./index.css";
import PropTypes from "prop-types";
const FooterCopyright = props => {
  return <div className={style.footer_copyright}>{props.children}</div>;
};
FooterCopyright.propTypes = {
  children: PropTypes.node
};
export default FooterCopyright;
