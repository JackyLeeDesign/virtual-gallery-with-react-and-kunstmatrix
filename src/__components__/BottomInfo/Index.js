import React from "react";
import FooterCopyright from "./FooterCopyright";
const BottomInfo = () => {
  return (
    <FooterCopyright>
      Copyright © 2023 JackyLee
      <br />
      建議瀏覽器：Chrome，Firefox，IE10.0 以上版本（螢幕最佳顯示效果為 1920 * 1280)
    </FooterCopyright>
  );
};
export default BottomInfo;
