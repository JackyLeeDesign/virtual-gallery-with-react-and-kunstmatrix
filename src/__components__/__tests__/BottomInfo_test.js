import { configure } from "enzyme";
import renderer from "react-test-renderer";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import React from "react";
import BottomInfo from "c:/Users/Jacky.lee/Desktop/MusicWeb/src/components/BottomInfo";
describe("BottomInfo:UI Test", () => {
  it("UI test", () => {
    const component = renderer.create(<BottomInfo />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
