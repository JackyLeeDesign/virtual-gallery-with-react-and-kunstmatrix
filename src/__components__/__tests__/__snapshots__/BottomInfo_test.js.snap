// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomInfo:UI Test UI test 1`] = `
<div
  className="footer_copyright"
>
  Copyright © 2019 JackyLee
  <br />
  建議瀏覽器：Chrome，Firefox，IE10.0以上版本（螢幕最佳顯示效果為1920 * 1280)
</div>
`;
